dockers/docker-lldp-sv2/reconfigure.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e
+
+num_of_interfaces=32
+if_step=4
+last_if_idx=$((num_of_interfaces*if_step - if_step))
+
+function wait_until_if_exists
+{
+  while /bin/true ;
+  do
+    if [ -L /sys/class/net/"$if" ];
+    then
+      break
+    fi
+    sleep 1
+  done
+  echo interface "$if" is up
+}
+
+
+function wait_until_if_not_exists
+{
+  while /bin/true ;
+  do
+    if [ ! -L /sys/class/net/"$if" ];
+    then
+      break
+    fi
+    sleep 1
+  done
+  echo interface "$if" is down
+}
+
+
+while /bin/true ;
+do
+  # wait until all interfaces are up
```

Comment:
Is it possible that some interface will never be up, then we wait forever here?

---
In reply to: [85829244](https://github.com/Azure/sonic-buildimage/pull/49#discussion_r85829244) [](ancestors = 85829244)